crates/types/src/replicated_loglet/params.rs

```diff
@@ -90,15 +91,12 @@ impl ReplicatedLogletId {
 pub struct NodeSet(#[serde_as(as = "HashSet<DisplayFromStr>")] HashSet<PlainNodeId>);
 
 impl Display for NodeSet {
+    /// The alternate format displays a *sorted* list of short-form plain node ids, suitable for human-friendly output.
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "[")?;
-        for (i, id) in self.0.iter().enumerate() {
-            if i != 0 {
-                write!(f, ", ")?;
-            }
-            write!(f, "{}", id)?;
+        match f.alternate() {
+            false => self.write_nodes(f),
+            true => self.write_nodes_sorted(f),
```

Comment:
imho, making those as members of the struct adds unnecessary noise to the type.

Reply:
it's a micro-nit though.

Reply:
Fixed!